### PR TITLE
Release/0.3.0 asset bundling

### PR DESCRIPTION
**Pull request by sibartlett**

### CODE REVIEW

File: app/bundles.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var bundles = require('./../bundles'),
+    settings = require('./config'),
+    compiledBundles = {};
+
+if (settings.env === 'production') {
+
+    Object.keys(bundles).forEach(function(key) {
```
Comment (hhaidar) [inline app/bundles.js]: A normal human being would use underscore/lodash, but not @sibartlett

File: app/controllers/account.js [hunk 2]
```diff
@@ -25,7 +26,8 @@ module.exports = function() {
     app.get('/', middlewares.requireLogin, function(req, res) {
         res.render('chat.html', {
             account: req.user.toJSON(),
-            settings: settings
+            settings: settings,
+            bundles: bundles
```
Comment (hhaidar) [inline app/controllers/account.js]: Can we make it a nunjucks global?

File: bundles.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+module.exports = {
```
Comment (hhaidar) [inline bundles.js]: It'd be nice if this was a yaml file

File: media/js/chat.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+window.client = new window.LCB.Client();
```
Comment (hhaidar) [inline media/js/chat.js]: this code is in chat.html, do we need this file?

Comment (sibartlett) [inline media/js/chat.js]: I removed it from chat.html

@geoffwhittington was saying that inline JavaScript impacts Content Security Policy

**Pull request merged**